app/scripts/controllers/attestation.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+/**
+ * @ngdoc function
+ * @name ecampusApp.controller:AttestationCtrl
+ * @description
+ * # AttestationCtrl
+ * Controller of the ecampusApp
+ */
+angular.module('ecampusApp')
+  .controller('AttestationCtrl', ['$scope', 'Api', function ($scope, Api) {
+
+     $scope.attestNumTyped = '';
+
+    function LoadYears() {
+      var url = 'Attestation/studyYear';
+      Api.execute("GET", url)
+        .then(function (response) {
+            $scope.Years = response;
+          },
+          function (result) {
```

Comment:
I saw an example in official documentation
`// Simple GET request example:
$http({
  method: 'GET',
  url: '/someUrl'
}).then(function successCallback(response) {
    // this callback will be called asynchronously
    // when the response is available
  }, function errorCallback(response) {
    // called asynchronously if an error occurs
    // or server returns response with an error status.
  });`
and add second function handler if error occurs